.github/workflows/publish-next-release-unstable.yml

```diff
@@ -0,0 +1,73 @@
+# Description: This workflow runs unit + e2e tests, then publishes UI packages
+#              to `@next-release` NPM tag.
+#
+# Triggered by: merge to `next-release/main`
+
+name: Test and Publish / next-release-unstable
+
+on:
+  push:
+    branches: [next-release/main]
+
+permissions:
+  id-token: write # This is required for requesting the JWT
+  contents: read # This is required for actions/checkout
+
+jobs:
+  setup-cache:
+    uses: aws-amplify/amplify-ui/.github/workflows/reusable-setup-cache.yml@next-release/main
+    with:
+      commit: ${{ github.sha }}
+      repository: ${{ github.repository }}
+
+  publish:
+    uses: ./.github/workflows/reusable-tagged-publish.yml
+    with:
+      dist-tag: next-release-unstable
```

Comment:
This publishes the latest of `next-release` branch to a tag `next-release-unstable` without waiting for tests.